config/default.yml

```diff
@@ -95,6 +95,9 @@ Betterment/ServerErrorAssertion:
   Include:
     - spec/requests/**/*_spec.rb
 
+Betterment/SimpleDelegator:
+  SafeAutoCorrect: false
```

Comment:
I think this should be `AutoCorrect: false`, since it doesn't support autocorrection.

Reply:
updated!